### ISSUE TITLE
Create examples.md

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/mean-share-and-balance#308]

**Comment by amanda-phet** [comment]
Still very much a draft, but the file is created and has a couple things populated.

**Comment by jbphet** [comment]
Looks great, I've just got a few comments below.  This review turned out to be quite beneficial because it revealed a number of minor bugs, most of which @marlitas has already addressed.

Here are my comments:

- The term "people" is used in a number of places as the thing for which values can be set, for example, "These properties can be used to get or set the data values for individual cups/people/balls."  However, the API uses the term "plates".  Should we use "plates" in the text for greater consistency?
- There is a sentence that says, "The visibilily of the Kick button is determined by the last ball being kicked."  This doesn't sound quite accurate to me.  It's determined by whether there are any more balls available to be kicked.  I'd suggest rewording this.
- The sentence "To determine if the notepad reflects the mean will depend on the screen's context" feels awkward.  How about something like, "Determining whether the mean is being displayed on a screen is done a bit differently on each of the four screens.  See the details below."
- The information about determining whether the mean is displayed on the fourth screen doesn't seem quite complete.  Do we want to count the case where the user has selected the fixed fulcrum (see screenshot below)?  If so, we'd need to add the case where `meanShareAndBalance.balancePointScreen.model.meanFulcrumFixedProperty` is `true` to the list. ![image](https://github.com/phetsims/mean-share-and-balance/assets/3082831/0954d819-686d-41c3-b75e-83503fe2ff7e)
- It took me a while to understand the workings of the array in the "Probability by Distance" section because I'm used to thinking of probabilities as values between 0 and 1.  I figured it out after experimenting with it, but I noticed that the source code uses the term "weights" and "weighted values", which made a lot of sense to me.  Perhaps we should use those terms in the explanatory text here too.
